.npmignore

```diff
@@ -1 +1,34 @@
 node-tests/
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+FEATURES.md
+STYLEGUIDE.md
+VERSION
+bin
+blueprints
+bower.json
+bower_components
+config
+# In the future we will likely want to restrict exactly
+# which parts of `/dist` we want to include.
+#
+# dist
```

Comment:
I feel like we should split this out to the very bottom of the file. Especially since it's commented to begin with. It can just be presented as a documentation of:

> Currently we include the entire `dist` folder, but at some point we may wish to be more selective about what we include.